src/config/sidebar.js

```diff
@@ -11,5 +11,9 @@ export default {
   profile: true,
   account: true,
   highlight: true,
-  icons: true
+  icons: true,
+  customSideBarComponent: {
```

Comment:
This should be `customSideBarFooterComponent` as there will be a header one in the future.